example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
What is this for?

Reply:
This directory got added by TSDX and was discussed how it would be a small playground app where one can test out their component(s).  This is its own conversation on how to best use it so I left it alone but will discuss and add a "how-to" for this in a separate ticket.